lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -1,38 +1,56 @@
 .adf {
     &-metadata-properties {
-        .mat-expansion-panel-header.mat-expanded:hover,
-        .mat-expansion-panel-header.mat-expanded:focus {
-            background: var(--adf-theme-background-hover-color);
-        }
-
         mat-expansion-panel-header {
-            height: 64px;
+            height: 56px;
 
             .adf-metadata-properties-title {
-                font-weight: normal;
+                font-weight: 700;
                 font-size: 15px;
+                padding-left: 12px;
             }
         }
 
         .mat-expansion-panel:not([class*='mat-elevation-z']) {
             box-shadow: none;
         }
 
-        .adf-metadata-properties-tag {
-            height: 40px;
+        .adf-mat-divider {
+            margin-left: -24px;
+            margin-right: -24px;
+        }
+
+        .adf-edit-icon-buttons {
+            color: var(--adf-theme-foreground-text-color-054);
```

Comment:
This color is different that the one for `close` icon next to it, is it desired behaviour? Also this color is different than the one used for edit icons inside the panel itself which looks weird, can we unified them?
<img width="801" alt="Zrzut ekranu 2023-09-29 o 11 12 02" src="https://github.com/Alfresco/alfresco-ng2-components/assets/113341662/15e5dcc0-3f4e-4a38-a65e-b2dc1b639af4">

Reply:
Also you can see that icons in the header and inside panel are misaligned which doesn't look good

Reply:
@MichalKinas The edit icon next to the header as per the Figma design and the one inside the panel itself will go away from the panel. In Anukriti PR we are removing this edit icon inside the panel as per the Figma design.

Reply:
@rbahirsheth please provide to Michał link to PR you are mentioning.

Reply:
This is the PR link: https://github.com/Alfresco/alfresco-ng2-components/pull/8898 @rbahirsheth mentioned it

Reply:
This is the ACA PR
https://github.com/Alfresco/alfresco-content-app/pull/3415

Reply:
@MichalKinas This has been addressed below the PR link, It will be reviewed and merged after this PR. So request you to resolve this comment.
https://github.com/Alfresco/alfresco-ng2-components/pull/8898